tasks/main.yml

```diff
@@ -14,6 +14,11 @@
     - "{{ prometheus_configuration_directory }}/prometheus/targets"
     - "{{ prometheus_configuration_directory }}/prometheus/rules"
 
+- name: Get status of {{ prometheus_custom_configuration_file }}
+  stat:
+    path: "{{ prometheus_custom_configuration_file }}"
+  register: result
```

Comment:
This has to be delegated to the deployment/manager node. When the prometheus node != deployment/manager node this check will not work because the configuration directory is only available on the deployment/manager node, not on the prometheus node.